plugin.video.sms/resources/lib/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
you might want to use a spdx based license header in your python files. See for instance the kodi core: https://github.com/xbmc/xbmc/blob/master/xbmc/profiles/ProfileManager.cpp#L1-L7